src/clj/puppetlabs/services/protocols/versioned_code.clj

```diff
@@ -5,4 +5,7 @@
 
   (current-code-id
    [this environment]
-   "Returns the current code id (representing the freshest code) for the given environment."))
+   "Returns the current code id (representing the freshest code) for the given environment.")
+  (get-code-content
+   [this environment code-id file-path]
```

Comment:
When they're getting a file out of code-dir that's not in an environment. I think I might be :confused: about something